stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/ChatInfoItem.kt

```diff
@@ -10,92 +10,45 @@ sealed class ChatInfoItem {
 
     val id: String
         get() = when (this) {
-            is MemberItem -> member.getUserId()
+            is MemberItem -> chatMember.member.getUserId()
+            is Option -> optionType.name
+            is StatefulOption -> optionType.name
             else -> this::class.java.simpleName
         }
 
-    data class MemberItem(val member: Member) : ChatInfoItem()
+    data class MemberItem(val chatMember: ChatMember) : ChatInfoItem()
+    data class MembersSeparator(val membersToShow: Int) : ChatInfoItem()
+    data class ChannelName(val name: String) : ChatInfoItem()
     object Separator : ChatInfoItem()
+    data class Option(val optionType: OptionType) : ChatInfoItem()
+    data class StatefulOption(val optionType: OptionType, val isChecked: Boolean) : ChatInfoItem()
+}
 
-    sealed class Option : ChatInfoItem() {
```

Comment:
It was hard to reuse the old implementation so I've decided to convert different options into enum class to make it more reusable